docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -303,9 +303,68 @@ Continue configuring your Kubernetes provider by clicking **Add Region** and com
     ```yml
     istioCompatibility: enabled: true
     ```
+    
+  - Overrides to publish Node-IP as the server broadcast address.
+  
+    By default, master and T-Server pod IPs are published as the server broadcast address. To publish the IPs of the nodes on which database pods are deployed, add the following YAML to each zone override configuration:
+  
+    ```yml
+    master:
+      podAnnotations:
+        prometheus.io/path: "/prometheus-metrics"
+        # Any other required annotations
+    
```

Comment:
This whole section can be removed as well. Same as the suggestion given by Jason for tserver section.

Reply:
Removed